fe/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -2359,7 +2359,7 @@ public void createDb(CreateDbStmt stmt) throws DdlException {
                 Database db = new Database(id, fullDbName);
                 db.setClusterName(clusterName);
                 unprotectCreateDb(db);
-                editLog.logCreateDb(db);
+                Catalog.getInstance().getEditLog().logCreateDb(db);
```

Comment:
Is this needed to change?

Reply:
It's order to make the test for `catalog.createDb` and `catalog.dropDb`  write easily.

Reply:
I think it isn't  a good idea to use `getInstance()` here, because this may lead some error if another `Catalog` exist in system, and it is strange to call `getInstance()` to get itself.

In addition I think it is better to limit use of singleton as little as possible. 

So, could you find another way to write unit test?

Reply:
OK.